extensions/amp-mustache/0.1/amp-mustache.js

```diff
@@ -33,14 +36,37 @@ export class AmpMustache extends AMP.BaseTemplate {
 
   /** @override */
   compileCallback() {
+    /** @private @const {!JsonObject} */
+    this.nestedTemplates_ = dict();
```

Comment:
This needs to be declared in the constructor.

Reply:
This method is called directly from within the constructor, @choumx and I tried to find a workaround but stuck with this eventually